lib/SymbolGraphGen/SymbolGraph.cpp

```diff
@@ -98,6 +95,7 @@ PrintOptions SymbolGraph::getDeclarationFragmentsPrintOptions() const {
   // In "emit modules separately" jobs, access modifiers show up as attributes,
   // but we don't want them to be printed in declarations
   ExcludeAttrs.insert(std::make_pair("DAK_AccessControl", DAK_AccessControl));
+  ExcludeAttrs.insert(std::make_pair("DAK_SetterAccess", DAK_SetterAccess));
```

Comment:
We should probably clean up printing at some point. We assert that `DAK_SetterAccess` is in the exclude list if `DAK_AccessControl` is, but I don't really see any reason to do that. That printing method also don't check `UserInaccessible`.